front/src/pages/teachers/teachers-information/index.js

```diff
@@ -65,7 +80,7 @@ class TeachersInformation extends Component {
                 <Groups>{`${groupsAmount} groups`}</Groups>
                 <Students>{`${studentsAmount} students`}</Students>
               </p>
-              <Button type="primary">Change password</Button>
+              <Button data-id={id} onClick={this.showModalChangePassword} type="primary">Change password</Button>
```

Comment:
To pass id you can define `onClick` in the format `() => this.showModalChangePassword(id)`, it is much easier to understand

Reply:
Done